sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -179,7 +179,7 @@ namespace Azure { namespace Core {
     {
       // In the cases where the customer doesn't want to use a context we new one up and pass it
       // through
-      return Poll(Context::ApplicationContext);
+      return Poll({});
```

Comment:
Something we should look at later, How do I pass a span into a poll since I can't pass a context?

Reply:
What do you mean "pass a span"? If you're using OpenTelemetry, that's handled by the C++ OpenTelemetry library. I'm not sure of any other "span" construct.

And in this case, I'm making the code match the comment - the comment indicates that the code should accept a *new* context, not the ApplicationContext.